### PR TITLE
Update composer.json

### PR DESCRIPTION
**Pull request by enumag**
Don't merge yet, I'd like to remove Tracy from dev dependencies as well which is not currently possible because of Kdyby/Translation.

### CODE REVIEW

**Comment by enumag** [comment]
Travis failed because of https://github.com/Kdyby/TesterExtras/blob/master/bin/composer-nette.php#L106. What is the reason for that error?

File: composer.json [hunk 1]
```diff
@@ -34,26 +33,10 @@
 		"kdyby/translation": "~2.0@dev"
 	},
 	"require-dev": {
-		"nette/application": "~2.3@dev",
```
Comment (fprochazka) [inline composer.json]: please don't remove this

Comment (enumag) [inline composer.json]: May I ask why? :-)

**Comment by fprochazka** [comment]
@enumag actually we need all of them in there. If there was only one or two that are really needed, the script would have it really hard to enforce what versions are installed. If we'd depend only on `nette/application: @dev`, there would be no easy way to install also it's nette dependencies in dev version. I've aleady encountered a situation, where the packages just didn't work together at all and this is the only sane way to handle it.

**Comment by enumag** [comment]
@fprochazka Huh? Why not just use the minimum-stability option?

**Comment by enumag** [comment]
Anyway I've removed all changes to the require-dev section. The rest should be fine to merge I think.

**Comment by fprochazka** [comment]
@enumag minimum stability affects all the dependencies, not only Nette.

**Comment by fprochazka** [comment]
:+1: thanks

**Pull request merged**